src/program.ts

```diff
@@ -830,7 +830,7 @@ export class Program extends DiagnosticEmitter {
           if (!name.length) continue; // explicitly disabled
           let firstChar = name.charCodeAt(0);
           if (firstChar >= CharCode._0 && firstChar <= CharCode._9) {
-            this.registerConstantInteger(alias, Type.i32, i64_new(parseI32(name, 10)));
+            this.registerConstantInteger(alias, Type.i32, i64_new(parseInt(name, 10)));
```

Comment:
When compiling to WASM this will fail because `parseInt` returns `f64` but `i64_new` expects `i32`. A portable `I32.parseInt` would be ideal I guess, otherwise an explicit cast.

Reply:
yeah, good catch!